### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by marchitelli**
Italian locale creation for plugin PDF.JS

### CODE REVIEW

File: locale/it_IT/locale.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/pdfJsViewer/locale/en_US/locale.xml
+  *
+  * Copyright (c) 2013-2014 Simon Fraser University Library
```
Comment (asmecher) [inline locale/it_IT/locale.xml]: Mind changing the end dates to 2015?

**Comment by marchitelli** [comment]
chaged!

File: locale/it_IT/locale.xml (hunk 1 above)
Comment (marchitelli) [inline locale/it_IT/locale.xml]: changed

**Comment by asmecher** [comment]
Merged. Thanks, Andrea!

**Pull request merged**